app/controllers/prisoner_details_controller.rb

```diff
@@ -23,7 +30,8 @@ def update
   def prisoner_params
     dob = [:'date_of_birth(3i)', :'date_of_birth(2i)', :'date_of_birth(1i)']
     if params[:date_of_birth_native].present?
-      params[:prisoner][:date_of_birth] = Date.parse(params[:date_of_birth_native])
+      params[:prisoner][:date_of_birth] = Date.parse(
+        params[:date_of_birth_native])
```

Comment:
I think this kind of thing works better as

```ruby
params[:prisoner][:date_of_birth] =
  Date.parse(params[:date_of_birth_native])
```

because it keeps the parameters with the method.